src/vs/editor/contrib/inlineEdit/browser/inlineEditSideBySideWidget.ts

```diff
@@ -281,13 +296,13 @@ class InlineEditSideBySideContentWidget extends Disposable implements IOverlayWi
 		this._register(this._previewEditorObs.setDecorations(this._modifiedDecorations));
 
 		this._register(autorun(reader => {
+			this._previewEditorObs.model.read(reader);
```

Comment:
I'm not sure you need this dependency

Reply:
Yeah, I don't it was leftover after some experimentation. Deleted.